cabal.project

```diff
@@ -178,8 +178,8 @@ source-repository-package
 -- of cardano-node are also updated.
 source-repository-package
   type: git
-  location: https://github.com/input-output-hk/cardano-node.git
-  tag: ed11e8b6429d4af1cb2539460e5cb2283a06b2dc
+  location: https://github.com/raduom/cardano-node.git
+  tag: eef8df22d8a982814b336507716c19b7fc851e39
```

Comment:
is there a PR on `cardano-node` with your changes? We should switch back to the main repo as soon as possible